### PR TITLE
Added Francisco's info under section C

### PR DESCRIPTION
**Pull request by Francisco-Ibarra07**
I added my information under section C for this week's deliverable. 
Note: I use VSCode and I have auto-format every time I save a file. I guess that's why Git is saying that I changed a whole bunch of lines even though I only worked on a few. I'll look into fixing that!

### CODE REVIEW

File: src/main/webapp/aboutus.html [hunk 1]
```diff
@@ -16,42 +16,41 @@
...
+<ul>
+  <li>Summer Feelz: </li>
+  <li>Aspirational Hobby: </li>
+  <li>Ask me About: </li>
+</ul>
+<h2>Francisco Ibarra</h2>
+<ul>
+  <li><u>Summer Feelz:</u> It's finally here!</li>
+  <li><u>Aspirational Hobby:</u> If I had all the time in the world, I would totally get into competitive ESports! I
+    grew up
```
Comment (abisolaOlawale) [inline src/main/webapp/aboutus.html]: This probably has to do with the formatting issues you mentioned with VScode, and it's super minor, but consider removing the extra whitespace between "grew up" and the rest of your sentence

**Review by abisolaOlawale — CHANGES_REQUESTED** [review]
minor whitespace issue on line 49

**Review by abisolaOlawale — APPROVED** [review]
Looks good to me!

File: src/main/webapp/aboutus.html (hunk 1 above)
Comment (Francisco-Ibarra07) [reply src/main/webapp/aboutus.html]: Fixed!

**Pull request merged**